data/human/paradise world jobs.txt

```diff
@@ -578,6 +578,105 @@ mission "Paradise Job: Wilderness Retreat"
 		dialog `Your passengers are met by an overly enthusiastic greeting service of comically overdressed "wilderness guides." You can't help but think these people are either about to end up on a popular reality show, or be safely fleeced of their money, but they're not the sort of characters who engender much sympathy for such fates. One of the greeters quietly transfers to you your payment of <payment>.`
 
 
+mission "Gemini Shipyards Shipment [0]"
+	name "Shipment to <planet>"
+	description "Deliver <cargo> to <destination>, the home of Gemini Shipyards, a constituent shipbuilding company of the Republic Navy Yard and one of the most advanced shipyards in human space. Payment is <payment>."
+	cargo "weapons" 5 2 .1
+	job
+	repeat
+	to offer
+		random < 15
+		or
+			not "war begins"
+			and
+				has "war begins"
+				has "geminus rebuilt"
+	source
+		near Castor 2 10
+		attributes factory
+	destination Geminus
+	npc
+		government Pirate
+		personality heroic harvests plunders
+		fleet "Small Northern Pirates"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 0 160
+		dialog phrase "generic cargo delivery payment"
+
+mission "Gemini Shipyards Shipment [1]"
+	name "Shipment to <planet>"
+	description "Deliver <cargo> to <destination>, the home of Gemini Shipyards, a constituent shipbuilding company of the Republic Navy Yard and one of the most advanced shipyards in human space. Payment is <payment>."
+	cargo "ship systems" 5 2 .1
+	job
+	repeat
+	to offer
+		random < 15
+		or
+			not "war begins"
+			and
+				has "war begins"
+				has "geminus rebuilt"
+	source
+		near Castor 2 10
+		attributes factory
+	destination Geminus
+	npc
+		government Pirate
+		personality heroic harvests plunders
+		fleet "Small Northern Pirates"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 0 160
+		dialog phrase "generic cargo delivery payment"
+
+mission "Lovelace Labs Shipment [0]"
+	name "Shipment to <planet>"
+	description "Deliver <cargo> to <destination>, the home of Lovelace Labs, a well-respected weapons manufacturer in the region. Payment is <payment>."
+	cargo Metal 5 2 .1
+	job
+	repeat
+	to offer
+		random < 10
+	source
+		near Aldebaran 2 10
+		attributes core factory
+	destination Ada
+	npc
+		government Pirate
+		personality heroic harvests plunders
+		fleet "Small Northern Pirates"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 0 160
+		dialog phrase "generic cargo delivery payment"
+
+mission "Vinci Nanocircuits Shipment [0]"
+	name "Shipment to <planet>"
+	description "Deliver <cargo> to <destination>, the home of Vinci Nanocircuits, one of the largest electronics firms in human space. Payment is <payment>."
+	cargo Electronics 5 2 .1
+	job
+	repeat
+	to offer
+		random < 10
+	source
+		near Miaplacidus 2 10
+		attributes core factory
+	destination Vinci
+	npc
```

Comment:
Why pirates on this mission? The other missions have the reasoning that the pirates are maybe getting material to upgrade their ships, and Vinci is smack dab in the middle of the Paradise Worlds.

Reply:
Not sure, I added this mission something like 8 months ago. I'll change it.